core/tests/snapshots.rs

```diff
@@ -169,22 +178,16 @@ mod tests {
                 .as_ref()
                 .unwrap()
                 .snapshot_path,
-            snapshot_utils::build_full_snapshot_archive_path(
-                snapshot_package_output_path.to_path_buf(),
-                old_last_bank.slot(),
-                &old_last_bank.get_accounts_hash(),
-                ArchiveFormat::TarBzip2,
-            ),
+            &full_snapshot_archive_info,
             None,
-            ArchiveFormat::TarBzip2,
             old_genesis_config,
             None,
             None,
             AccountSecondaryIndexes::default(),
             false,
             None,
             accounts_db::AccountShrinkThreshold::default(),
-            check_hash_calculation,
+            false,
```

Comment:
The variable `check_hash_calculation` is defined above as the constant `false`, so I thought this would be more clear.

Here's the line in particular:
https://github.com/solana-labs/solana/blob/e4be00fece487d5413f7569b2e46c770b9b4d12e/core/tests/snapshots.rs#L163

Do you have a preference? I've found that a lot of the tests just use the constants instead of creating variables for each one.

Reply:
For me, Boolean value is always confusing, true for what, false for what? You do not know it until you check out the function signature -- and worse for return values: the implementation. So I always prefer some variables. But I also agree we have the hard coded one everywhere -- as it is easier to write -- so it is not you who is initiating it -- but just saw the change and thus the comment.

Reply:
yeah let's keep the named variables where possible, this is also the benefit of having argument structs instead of long lists of arguments.

Reply:
Got it. Done!